diff --git a/node_modules/react-native-reanimated/Common/cpp/NativeModules/NativeReanimatedModule.cpp b/node_modules/react-native-reanimated/Common/cpp/NativeModules/NativeReanimatedModule.cpp
index a1ace04..d773825 100644
--- a/node_modules/react-native-reanimated/Common/cpp/NativeModules/NativeReanimatedModule.cpp
+++ b/node_modules/react-native-reanimated/Common/cpp/NativeModules/NativeReanimatedModule.cpp
@@ -3,7 +3,8 @@
 #ifdef RCT_NEW_ARCH_ENABLED
 #include <react/renderer/uimanager/UIManagerBinding.h>
 #include <react/renderer/uimanager/primitives.h>
-#if REACT_NATIVE_MINOR_VERSION >= 73 && defined(RCT_NEW_ARCH_ENABLED)
+#if REACT_NATIVE_MINOR_VERSION >= 73 && defined(RCT_NEW_ARCH_ENABLED) && __has_include(<react/utils/CoreFeatures.h>)
+#define REA_HAS_COREFEATURES 1
 #include <react/utils/CoreFeatures.h>
 #endif
 #endif
@@ -37,7 +38,7 @@
 
 using namespace facebook;
 
-#if REACT_NATIVE_MINOR_VERSION == 73 && defined(RCT_NEW_ARCH_ENABLED)
+#if REACT_NATIVE_MINOR_VERSION == 73 && defined(RCT_NEW_ARCH_ENABLED) && defined(REA_HAS_COREFEATURES)
 // Android can't find the definition of this static field
 bool CoreFeatures::useNativeState;
 #endif
diff --git a/node_modules/react-native-reanimated/apple/REAModule.h b/node_modules/react-native-reanimated/apple/REAModule.h
index 62dd42e..7215d59 100644
--- a/node_modules/react-native-reanimated/apple/REAModule.h
+++ b/node_modules/react-native-reanimated/apple/REAModule.h
@@ -1,9 +1,10 @@
 #ifdef RCT_NEW_ARCH_ENABLED
 #import <React/RCTInitializing.h>
-#if REACT_NATIVE_MINOR_VERSION >= 74
+#if REACT_NATIVE_MINOR_VERSION >= 74 && __has_include(<React/RCTRuntimeExecutorModule.h>)
+#define REA_HAS_RUNTIME_EXECUTOR_MODULE 1
 #import <React/RCTRuntimeExecutorModule.h>
 #import <ReactCommon/RCTRuntimeExecutor.h>
-#endif // REACT_NATIVE_MINOR_VERSION >= 74
+#endif // condicional REA_HAS_RUNTIME_EXECUTOR_MODULE
 #import <rnreanimated/rnreanimated.h>
 #else // RCT_NEW_ARCH_ENABLED
 #import <React/RCTBridgeModule.h>
@@ -21,9 +22,9 @@
 #ifdef RCT_NEW_ARCH_ENABLED
                        <NativeReanimatedModuleSpec,
                         RCTInitializing,
-#if REACT_NATIVE_MINOR_VERSION >= 74
+#if defined(REA_HAS_RUNTIME_EXECUTOR_MODULE)
                         RCTRuntimeExecutorModule,
-#endif // REACT_NATIVE_MINOR_VERSION >= 74
+#endif // REA_HAS_RUNTIME_EXECUTOR_MODULE
 #else
                        <RCTBridgeModule,
 #endif // RCT_NEW_ARCH_ENABLED
diff --git a/node_modules/react-native-reanimated/apple/REAModule.mm b/node_modules/react-native-reanimated/apple/REAModule.mm
index 4148ea5..502abef 100644
--- a/node_modules/react-native-reanimated/apple/REAModule.mm
+++ b/node_modules/react-native-reanimated/apple/REAModule.mm
@@ -63,9 +63,9 @@ typedef void (^AnimatedOperation)(REANodesManager *nodesManager);
 }
 
 @synthesize moduleRegistry = _moduleRegistry;
-#if REACT_NATIVE_MINOR_VERSION >= 74 && defined(RCT_NEW_ARCH_ENABLED)
+#if defined(REA_HAS_RUNTIME_EXECUTOR_MODULE) && defined(RCT_NEW_ARCH_ENABLED)
 @synthesize runtimeExecutor = _runtimeExecutor;
-#endif // REACT_NATIVE_MINOR_VERSION >= 74 && defined(RCT_NEW_ARCH_ENABLED)
+#endif // REA_HAS_RUNTIME_EXECUTOR_MODULE
 
 RCT_EXPORT_MODULE(ReanimatedModule);
 
@@ -283,6 +283,7 @@ RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(installTurboModule : (nonnull NSString *)
 #if REACT_NATIVE_MINOR_VERSION >= 74 && defined(RCT_NEW_ARCH_ENABLED)
     RCTCxxBridge *cxxBridge = (RCTCxxBridge *)self.bridge;
     auto &rnRuntime = *(jsi::Runtime *)cxxBridge.runtime;
+#if defined(REA_HAS_RUNTIME_EXECUTOR_MODULE) && defined(RCT_NEW_ARCH_ENABLED)
     auto executorFunction = ([executor = _runtimeExecutor](std::function<void(jsi::Runtime & runtime)> &&callback) {
       // Convert to Objective-C block so it can be captured properly.
       __block auto callbackBlock = callback;
@@ -291,6 +292,11 @@ RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(installTurboModule : (nonnull NSString *)
         callbackBlock(runtime);
       }];
     });
+#else
+    auto executorFunction = ([](std::function<void(jsi::Runtime & runtime)> &&callback) {
+      callback(rnRuntime);
+    });
+#endif // REA_HAS_RUNTIME_EXECUTOR_MODULE
     auto nativeReanimatedModule = reanimated::createReanimatedModuleBridgeless(
         _moduleRegistry, rnRuntime, std::string([valueUnpackerCode UTF8String]), executorFunction);
     [self attachReactEventListener];
